com.unity.netcode.gameobjects/Runtime/Logging/NetworkLog.cs

```diff
@@ -51,35 +51,58 @@ public static class NetworkLog
         /// <param name="message">The message to log</param>
         public static void LogErrorServer(string message) => LogServer(message, LogType.Error);
 
+        internal static NetworkManager NetworkManagerOverride;
```

Comment:
is this strictly for tests?

Reply:
It is indeed.  Since it was always using the singleton, up until this we have never fully tested server network logs generated clients.